src/components/Menu/Menu.types.ts

```diff
@@ -1,27 +1,30 @@
-import { ButtonProps } from '../Button';
 import { ListProps } from '../List';
-
-export interface MenuItem
-    extends Omit<ButtonProps, 'disruptive' | 'onClick' | 'value'> {
-    /**
-     * Value of the menu item
-     */
-    value: any;
-}
+import { MenuItemProps } from './MenuItem/MenuItem.types';
 
 export enum MenuType {
     disruptive = 'disruptive',
-    default = 'default',
+    primary = 'primary',
     neutral = 'neutral',
 }
 
+export enum MenuSize {
+    large = 'large',
+    medium = 'medium',
+    small = 'small',
+}
+
 export interface MenuProps
```

Comment:
Should there be a `multiple?: boolean` prop? When this prop, or will the MenuItem input type passed to the component determine this functionality? Would be good to see an updated story to get a better idea.